plugins/network-elements/nuage-vsp/src/com/cloud/network/manager/NuageVspManagerImpl.java

```diff
@@ -122,13 +213,13 @@
         cmdList.add(DeleteNuageVspDeviceCmd.class);
         cmdList.add(ListNuageVspDevicesCmd.class);
         cmdList.add(IssueNuageVspResourceRequestCmd.class);
-
+        cmdList.add(UpdateNuageVspDeviceCmd.class);
```

Comment:
Consider using ``com.google.guava.collect.Lists#newHashMap`` for a more succinct way to build the list.